test/Misc/serialized-diagnostics-batch-mode-nonprimary-errors.swift

```diff
@@ -0,0 +1,20 @@
+// Batch-mode has no good place to emit diagnostics that occur in non-primary files
+// because then cannot be locatlized to a particular primary's serialized diagnostics file.
+// When such an error occurs, a nonspecific error must be emitted
+// for the primary files so Xcode knows something happened.
+//
+// RUN: rm -f %t.*
+
+// RUN: not %target-swift-frontend -typecheck  -primary-file %s  -serialize-diagnostics-path %t.main.dia -primary-file %S/../Inputs/empty.swift  -serialize-diagnostics-path %t.empty.dia %S/Inputs/serialized-diagnostics-batch-mode-suppression-helper.swift  2> %t.stderr.txt
```

Comment:
This is no longer testing non-primary errors…but it still should! We need both tests.

Reply:
Yes, now testing both.